### PR TITLE
Improve cart

### PR DESCRIPTION
**Pull request by jbendzala**

### CODE REVIEW

File: lobis-repo/client/src/customer/components/ShoppingCart/ShoppingCartDrawer.tsx [hunk 1]
```diff
@@ -11,41 +11,34 @@ import {
...
 
-  const { data: cartData, isSuccess } = useGetCart({ enabled: !!auth });
+  const {
+    data: cartData,
+    isSuccess,
+    isLoading: isLoadingCart,
+  } = useGetCart({ enabled: !!auth });
   const [openDialog, setOpenDialog] = useState(false);
-  const [productInCartId, setProductInCartId] = useState(null);
+  const [cartItemId, setCartItemId] = useState(null);
```
Comment (cypcz) [inline lobis-repo/client/src/customer/components/ShoppingCart/ShoppingCartDrawer.tsx:36]: you should type the state variable like `useState<string | null>(null)`

**Review by cypcz — APPROVED** [review]

File: lobis-repo/client/src/customer/components/ShoppingCart/ShoppingCartDrawer.tsx [hunk 2]
```diff
@@ -55,33 +48,72 @@ export const ShoppingCartDrawer = () => {
...
+  };
+
+  const onMoreClick = ({ cartItemId, quantity }) => {
+    updateCartItem({
+      cartItemId,
+      quantity: quantity + 1,
+    });
+  };
+
+  cart?.sort(function (
```
Comment (jbendzala) [inline lobis-repo/client/src/customer/components/ShoppingCart/ShoppingCartDrawer.tsx:79]: @cypcz Would it make more sense to sort cartItems on BE?

Comment (cypcz) [reply lobis-repo/client/src/customer/components/ShoppingCart/ShoppingCartDrawer.tsx:79]: yeah sure, that would be better

Comment (jbendzala) [reply lobis-repo/client/src/customer/components/ShoppingCart/ShoppingCartDrawer.tsx:79]: Adding todo for now, need this changes for another MR

**Pull request merged**